### DISCUSSION TITLE
In windowed or full-screen mode, the program sound (music and sound effects) should be muted when the program is no longer in the "foreground" [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2610] [Ideas]
In windowed or full-screen mode, the program sound (music and sound effects) should be muted when the program is no longer in the "foreground", when the user changes programs, e.g. with the key combination "ALT" + "TAB", or by clicking on the window of another program.

**Comment by Branikolog** [comment]
I think, this thread is an appeal to our lead dev team members. ;) 
If this enhancement can be implemented? 
The only thing I disagree with @LeHerosInconnu is regarding "foreground" applications. As if player can accidentally miss main window and press surrounding workspace, which will cause "muting" in no necessary cases. 
My personal point is to mute program only if it was minimized into tray (with Alt+Tab from fullscreen or by pressing "_" button in the upper part of application in window mode).

**Reply by Branikolog** [reply]
I've created an issue, as this turned out to be a SDL2 problem.

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
